functions/src/ro/sci/service/CauciuSearchService.java

```diff
@@ -0,0 +1,26 @@
+package ro.sci.service;
+
+import ro.sci.domain.Cauciuc;
+import ro.sci.domain.CauciucRequest;
+
+public class CauciuSearchService {
```

Comment:
separation of concerns ii bun, dar gandeste-te daca nu cumva CauciucService ar tre sa stie si sa caute si sa salveze ?